src/Compilers/Core/Portable/CommandLine/CommonCompiler.cs

```diff
@@ -755,6 +755,13 @@ private int RunCore(TextWriter consoleOutput, ErrorLogger errorLogger, Cancellat
                             string finalRefPeFilePath = Arguments.OutputRefFilePath;
                             var refPeStreamProviderOpt = finalRefPeFilePath != null ? new CompilerEmitStreamProvider(this, finalRefPeFilePath) : null;
 
+                            RSAParameters? privateKeyOpt = null;
+                            if (compilation.Options.StrongNameProvider != null && compilation.Feature("BypassStrongName") == null)
```

Comment:
>BypassStrongName [](start = 103, length = 16)

Shouldn't these string values be in an enum somewhere?

Reply:
this appears to already exist in master. Found  another occurrence with the same pattern. :/

---
In reply to: [137086302](https://github.com/dotnet/roslyn/pull/21718#discussion_r137086302) [](ancestors = 137086302)

Reply:
I've removed this instance here, but there's still some uses in the `CSharpCompiler` and `VisualBasicCompiler` files.  Is there an enum of features somewhere that I should be adding this to?